### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by plata**
Enjoy!

### CODE REVIEW

File: i18n/Messages_fr.properties [hunk 1]
```diff
@@ -18,7 +18,7 @@ msgstr ""
 
 #: Applications/Games/League of Legends/BETA Client/script.js:41
 msgid "Select your region:"
-msgstr ""
+msgstr "Sélectionnez votre région :"
```
Comment (madoar) [inline i18n/Messages_fr.properties]: There seems to be a ` ` (space) character to much after `région`

Comment (plata) [reply i18n/Messages_fr.properties]: Can you fix it?

**Comment by plata** [comment]
How can this have conflicts?

**Comment by madoar** [comment]
@plata I have no clue

**Comment by madoar** [comment]
@plata any idea how we should handle the conflict in this PR?
I guess the easiest option is to manually resolve the conflicts, right?

**Comment by plata** [comment]
Yes. I did that already some time ago but I guess it must be merged once to avoid getting it over and over again. Are the translations up to date in Crowdin (at least for German)? If so, I would merge this.

**Comment by madoar** [comment]
I think so.
I haven't yet added some translations via Crowdin and as far as I know every change recently has been made via Crowdin, so from my side they should be up to date.

**Comment by plata** [comment]
Some translations are still missing. They should be added before this is merged.

**Pull request merged**